src/devices/uFire_ISE/UFirePh .cs

```diff
@@ -22,7 +22,7 @@ public class UFirePh : UFireIse
         public float Ph = 0;
```

Comment:
As I see you are using Ph = -1 for wrong values, then maybe initalize to -1?
Also consider using explicit property:
```csharp
public float Ph { get; set; } = -1;
```

Reply:
And also, rather than -1 for non valid, maybe better to use float.NaN?